sprints/sprint_2.html

```diff
@@ -108,6 +108,11 @@ <h2>Incident Response</h2>
       <br>
       <!-- The coverage was built upon <a href="https://github.com/oscd-initiative/Cortex-Analyzers" target="_blank">our fork</a> of repo with TheHive Analyzers and Responders, that we tagged with RE&CT IDs. -->
       That's why we've decided to focus on the TheHive Responders development this time. A detailed backlog on Incident Response could be found <a href="sprint_2_backlog.html" target="_blank">here</a>.
+      <br><br>
+      <h2>Results</h2>
+      - 242 new <a href="https://github.com/SigmaHQ/sigma/pull/1362">Sigma</a> rules added and 305 updated; <br>
```

Comment:
need to mention the OTR PR as well: https://github.com/SigmaHQ/sigma/pull/1158

Reply:
Done, thx